### PR TITLE
Run Prettier on JS code fences, part 2

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to https://github.com/mdn/content/pull/20713

The PR focuses only on JS code fences.

Idea is to gradually prettify all the JS code fences before the full automation.

### CODE REVIEW

**Comment by OnkarRuikar** [comment]
Ping @Josh-Cena

File: files/en-us/web/accessibility/aria/aria_live_regions/index.md [hunk 1]
```diff
@@ -181,8 +188,9 @@ As an illustration of `aria-atomic`, consider a site with a simple clock, showin
 /* basic JavaScript to update the clock */
 function updateClock() {
   const now = new Date();
-  document.getElementById('clock-hours').innerHTML = now.getHours();
-  document.getElementById('clock-mins').innerHTML = (`0${now.getMinutes()}`).substr(-2);
+  document.getElementById("clock-hours").innerHTML = now.getHours();
+  document.getElementById("clock-mins").innerHTML =
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/aria_live_regions/index.md:192]: Side fix:

```suggestion
  document.getElementById("clock-hours").textContent = now.getHours();
  document.getElementById("clock-mins").textContent =
```

File: files/en-us/web/accessibility/aria/attributes/aria-busy/index.md [hunk 2]
```diff
@@ -59,9 +59,9 @@ If changes to a rendered widget would create a state where the widget is missing
 ```
 
 ```js
-let el = document.getElementById('clock');
+let el = document.getElementById("clock");
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/attributes/aria-busy/index.md:62]:
```suggestion
const el = document.getElementById("clock");
```

File: files/en-us/web/accessibility/aria/roles/alert_role/index.md [hunk 3]
```diff
@@ -89,7 +89,9 @@ This snippet dynamically creates an element with an alert role and adds it to th
 ```js
 let myAlert = document.createElement("p");
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/alert_role/index.md:90]:
```suggestion
const myAlert = document.createElement("p");
```

File: files/en-us/web/accessibility/aria/roles/alert_role/index.md [hunk 4]
```diff
@@ -89,7 +89,9 @@ This snippet dynamically creates an element with an alert role and adds it to th
...
 myAlert.setAttribute("role", "alert");
-let myAlertText = document.createTextNode("You must agree with our terms of service to create an account.");
+let myAlertText = document.createTextNode(
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/alert_role/index.md:92]:
```suggestion
const myAlertText = document.createTextNode(
```

File: files/en-us/web/accessibility/aria/roles/button_role/index.md [hunk 5]
```diff
@@ -153,29 +153,33 @@ ul {
...
+
+  // Don't add blank entries to the list.
+  if (name.length > 0) {
+    listItem = document.createElement("li");
+    listItem.appendChild(document.createTextNode(name));
+
+    // Add the new name to the list.
+    let list = document.getElementById("nameList");
+    list.appendChild(listItem);
+  }
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/button_role/index.md:182]:
```suggestion
  const newNameInput = document.getElementById("newName");
  const name = newNameInput.value;
  newNameInput.value = ""; // clear the text field
  newNameInput.focus(); // give the text field focus to enable entering and additional name.

  // Don't add blank entries to the list.
  if (name.length > 0) {
    const listItem = document.createElement("li");
    listItem.appendChild(document.createTextNode(name));

    // Add the new name to the list.
    const list = document.getElementById("nameList");
    list.appendChild(listItem);
  }
```

File: files/en-us/web/accessibility/aria/roles/button_role/index.md [hunk 6]
```diff
@@ -236,15 +240,16 @@ function handleBtnClick(event) {
 
 function handleBtnKeyDown(event) {
   // Check to see if space or enter were pressed
-  if (event.key === " " || event.key === "Enter" || event.key === "Spacebar") { // "Spacebar" for IE11 support
+  if (event.key === " " || event.key === "Enter" || event.key === "Spacebar") {
+    // "Spacebar" for IE11 support
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/button_role/index.md]:
```suggestion
  // "Spacebar" is for IE11 support
  if (event.key === " " || event.key === "Enter" || event.key === "Spacebar") {
```

File: files/en-us/web/accessibility/aria/roles/radio_role/index.md [hunk 7]
```diff
@@ -210,12 +210,12 @@ A lot of JavaScript is required to make radio buttons out of non-semantic HTML.
 let radioGroups = document.querySelectorAll('[role="radiogroup"]');
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/radio_role/index.md:210]:
```suggestion
const radioGroups = document.querySelectorAll('[role="radiogroup"]');
```

File: files/en-us/web/accessibility/aria/roles/radio_role/index.md [hunk 8]
```diff
@@ -210,12 +210,12 @@ A lot of JavaScript is required to make radio buttons out of non-semantic HTML.
...
 
 for (let i = 0, groups = radioGroups.length; i < groups; i++) {
-  let radios = radioGroups[i].querySelectorAll('[role=radio]');
+  let radios = radioGroups[i].querySelectorAll("[role=radio]");
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/radio_role/index.md:213]:
```suggestion
  const radios = radioGroups[i].querySelectorAll("[role=radio]");
```

File: files/en-us/web/accessibility/aria/roles/switch_role/index.md [hunk 9]
```diff
@@ -115,17 +115,17 @@ The HTML is fairly simple here. The switch is implemented as a {{HTMLElement("bu
 This JavaScript code defines and applies a function to handle click events on switch widgets. The function changes the `aria-checked` attribute from `true` to `false`, or vice versa.
 
 ```js
-document.querySelectorAll(".switch").forEach(function(theSwitch) {
+document.querySelectorAll(".switch").forEach(function (theSwitch) {
   theSwitch.addEventListener("click", handleClickEvent, false);
 });
 
 function handleClickEvent(evt) {
   let el = evt.target;
```
Comment (teoli2003) [inline files/en-us/web/accessibility/aria/roles/switch_role/index.md:123]:
```suggestion
  const el = evt.target;
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**